sync-diff-inspector/sync-diff-inspector-overview.md

```diff
@@ -73,6 +73,9 @@ check-thread-count = 4
 # If enabled, SQL statements is exported to fix inconsistent tables.
 export-fix-sql = true
 
+# Only compares the data instead of the table structure, this config is an experimental feature.
```

Comment:
```suggestion
# Only compares the data instead of the table structure. This configuration item is an experimental feature. It is not recommended that you use it in the production environment.
```